tests/codegen/simd-intrinsic/simd-intrinsic-generic-load.rs

```diff
@@ -0,0 +1,36 @@
+//
```

Comment:
Could you also add a run-pass test which checks that masked loads and stores actually work independent of the codegen backend. Codegen tests only apply to the LLVM backend, not the GCC and Cranelift backends.